bin/i18n/package-lock.json

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
I'm curious what were the reasons we did not add this file until now?

Reply:
not sure! maybe @Hamms knows?

Reply:
speaking only for myself: laziness.

More specifically, because we only depend on internal dependencies here and because this isn't used in production, I'm not worried about some third-party package silently updating and causing unexpected inconsistencies between test and prod.

That being said, there's no reason _not_ to, so this LGTM